rollup.config.js

```diff
@@ -7,13 +7,28 @@ import copy from 'rollup-plugin-copy';
 import terser from '@rollup/plugin-terser';
 // import styles from "@ironkinoko/rollup-plugin-styles";
 
+/** @type {import('rollup').RollupOptions} */
 const config = {
     // The entry point for the bundler
-    input: 'www/js/app.js',
-    output: {
-        format: 'iife',
-        name: 'KiwixJSBundle'
+    // input: ['www/js/app.js', 'www/js/lib/library.js'],
+    input: {
+        bundle: 'www/js/app.js',
+        library: 'www/js/lib/library.js'
     },
+    output: [
+        {
+            dir: 'dist/www/js',
+            // file: 'bundle.js',
+            format: 'umd',
```

Comment:
To add use `library.js` we need to compile it using rollup too, so I have updated the config but it seems like the format `iife` and `umd` is not supported when compiling multiple files, I have limited knowledge when dealing with bundlers can you help a little here. Is this the right way to compile multiple files?

Reply:
@Rishabhg71  No, you don't want to compile this file. It's ONLY used (if I've understood correctly) by the html document you will load into the iframe. It's not used by the main application. Rollup config is only for the main application. You will just load this file directly, not as part of any bundle.

Reply:
I am trying that but seems like some of the features of js are not available in IE11 like async functions. In that case i will just legacy JS code

Reply:
For a small file like that, it's much easier just to write conformant code than to try to configure Babel. Note that Promises are not natively supported in IE11 or Firefox OS (another old one we still support), and so you may just have to use old-style callbacks if you are using anything asynchronous. However, not sure you really need it?

Reply:
Not only are promises missing, but also features like backticks. Nonetheless, I'll work with the writing lecgacy code.